core/ordering/cosipbft/pbft/mod.go

```diff
@@ -53,6 +55,29 @@ func (s State) String() string {
 	}
 }
 
+// defines prometheus metrics
+var (
+	promBlocks = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_blocks_total",
+		Help: "total number of blocks",
+	})
+
+	promTxs = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_transactions_block",
+		Help: "total number of transactions in the last block",
+	})
```

Comment:
the gauge is only going to be "collected" (i.e. sampled) once in a while by prometheus. We're going to miss a lot of values between samplings. It might make more sense to have a histogram here.

It might also make sense to have a counter, counting all transactions (independently of the block), but that would fit a separate need (measuring the level of "activity" in the system)